tests/_logs/README

```diff
@@ -0,0 +1,13 @@
+Pass --ptvsd-logs and/or --pydevd-logs to pytest to create log files here for a run.
+For example:
+
+    tox -e py37 -- --ptvsd-logs "tests/ptvsd/server/test_run.py::test_run[launch-file]"
+
+A separate ptvsd-{pid}.log file will be created for every ptvsd process spawned by
```

Comment:
Should we add a timestamp to the name?

Reply:
It would prevent them being overwritten, but make it hard to correlate ptvsd logs with pydevd logs.

I could add the process ID of the test worker, though, and also add it to the test worker log. That way you could at least find which pydevd log corresponds to which failed test.

Reply:
Ah no, that doesn't really work... because the same worker will run multiple tests. 

However, there's already existing logic that generates pydevd logs separately for every `debug.Session` in a temp directory - that's used to dump them if the test fails or times out. I can uncomment the line that logs the name of that temp file - then you can see it in test logs even for successful tests, and inspect it there.

For this switch, let's keep it as is. It's mainly for debugging a specific test, anyway. And once we have end-to-end logs, the problem goes away.

Reply:
For clarity, I was thinking `ptvsd-{pid}-{timestamp}.log`. We can do that change if it really becomes a issue.